src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/Licence.kt

```diff
@@ -13,7 +13,11 @@ data class Licence(
   @Schema(description = "The licence type", example = "AP")
   val licenceType: LicenceType,
 
-  @Schema(description = "The policy version number the licence is currently on", example = "2.1")
+  @Schema(
+    description = "The policy version number the licence is currently on",
+    example = "2.1",
+    allowableValues = ["1.0", "2.0", "2.1"],
+  )
   val policyVersion: String,
```

Comment:
mmh.. we could possibly move from a string over to an enum instance..

Reply:
Moved to enum instance